src/Orleans.Core/DistributedTracing/EventSourceEvents.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+
+namespace Orleans.DistributedTracing.EventSourceEvents
+{
+    [EventSource(Name = "Microsoft-Orleans-CallBackDataEvent")]
```

Comment:
Remove `Event` from the end of the name and consider a name like `"Microsoft-Orleans-Client"` & merge these two classes